node-intro/hello-node.ts

```diff
@@ -0,0 +1 @@
+console.log('Hello, Node');
```

Comment:
You should be logging `process.env` here as well. Did you commit before you added that?